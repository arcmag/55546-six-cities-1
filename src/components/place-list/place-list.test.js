import * as React from 'react';
import * as renderer from 'react-test-renderer';

import PlaceList from './place-list';

const mock = {
  selectedCity: `Paris`,
  offers: []
};

describe(`Test PlaceList`, () => {
  it(`Test PlaceList renderer`, () => {
    const tree = renderer.create(<PlaceList
      renderOffers={jest.fn()}
      onAddHotelInFavorite={jest.fn()}
      onSetActionCard={jest.fn()}
      offers={mock.offers}
      selectedCity={mock.selectedCity}
    />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
