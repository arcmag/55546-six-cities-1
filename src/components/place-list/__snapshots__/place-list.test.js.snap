// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test PlaceList Test PlaceList renderer 1`] = `
<div
  className="cities__places-list places__list tabs__content"
>
  <section
    className="cities__no-places"
  >
    <div
      className="cities__status-wrapper tabs__content"
    >
      <b
        className="cities__status"
      >
        No places to stay available
      </b>
      <p
        className="cities__status-description"
      >
        We could not find any property availbale at the moment in Dusseldorf
      </p>
    </div>
  </section>
</div>
`;
